src/any_agent/evaluation/agent.py

```diff
@@ -1,121 +1,85 @@
-import json
-
-from pydantic import BaseModel
+from collections.abc import Callable
+from typing import Any
 
 from any_agent import AgentConfig, AnyAgent
+from any_agent.config import AgentFramework
 from any_agent.evaluation.schemas import AgentOutput
+from any_agent.evaluation.tools import AgentEvaluationTools
 from any_agent.tracing.agent_trace import AgentTrace
+from any_agent.utils.asyncio_sync import run_async_in_sync
 
-MAX_EVIDENCE_LENGTH: int = 500
-
+AGENT_INSTRUCTIONS = f"""You are a helpful assistant that will be used to evaluate the correctness of an agent trace. Given a specific question regarding the quality of the something about the agent, utilize the appropriate tools in order to gather the answer needed in order to accurately answer the question. If you are asked anything about what the agent did, you should strongly consider using the get_evidence_from_spans tool to get the evidence. However, if the question is about specific details of the agent's actions, you don't necessarily need to use the get_evidence_from_spans tool.
```

Comment:
Do you have preliminary results of how this will work in a real invocation? It would be really good to check it out!!!

Reply:
Totally! Our ongoing work with evaluations should help to give us some good feedback about how this works.